manager/controllers/default/resource/update.class.php

```diff
@@ -119,7 +119,7 @@ public function process(array $scriptProperties = array()) {
         $this->resourceArray['cacheable'] = intval($this->resourceArray['cacheable']) == 1 ? true : false;
         $this->resourceArray['deleted'] = intval($this->resourceArray['deleted']) == 1 ? true : false;
         $this->resourceArray['uri_override'] = intval($this->resourceArray['uri_override']) == 1 ? true : false;
-        $this->resourceArray['syncsite'] = isset($this->resourceArray['syncsite']) && intval($this->resourceArray['syncsite']) == 1 ? true : $this->context->getOption('syncsite_default', 1, $this->modx->_userConfig);
+        $this->resourceArray['syncsite'] = intval($this->context->getOption('syncsite_default', 1, $this->modx->_userConfig)) == 1 ? true : false;
```

Comment:
Once again, this is just my understanding of the code : 

1. we grab the `modResource` object, and make it available as an [array](https://github.com/modxcms/revolution/blob/master/manager/controllers/default/resource/update.class.php#L110)
2. then, we check for [form customization rules](https://github.com/modxcms/revolution/blob/master/manager/controllers/default/resource/update.class.php#L111-112), which allow to "override" some fields values (`syncsite` being one fo those)
3. then, we [process](https://github.com/modxcms/revolution/blob/master/manager/controllers/default/resource/update.class.php#L114-121) the array data so it become usable in ExtJS

Again, my interpretation is that overridden values in FC should have an higher priority (be used/applied) than settings.